_overviews/scala3-book/ca-multiversal-equality.md

```diff
@@ -7,6 +7,7 @@ num: 64
 previous-page: ca-type-classes
 next-page: ca-implicit-conversions
 ---
+<span class="tag tag-inline">Scala 3 only</span>
 
 
 Previously, Scala had *universal equality*: Two values of any types could be compared with each other using `==` and `!=`.
```

Comment:
@julienrf this page only adds tabs to a few code examples, but now it might seem strange to put "scala 2 and 3" tabs on a "scala 3 only" page

Reply:
I think we should either put a single “Scala 3 Only” tab _everywhere_, or no code tabs at all since there is already the “Scala 3 only” tag at the top of the page. What do you think would be the best option?

Reply:
I wasn’t sure what to do with these code blocks. It’s a little unusual because the code blocks I skipped work with Scala 2 & 3. I’m inclined to put “Scala 3 Only” at the top of the page, and not put anything on the code blocks, but I know that isn’t 100% accurate.

Reply:
perhaps we can keep it as it is, but maybe change the "scala 3 only" tabs to a new label like "new in Scala 3"?

Reply:
@alvinj I think that’s a good suggestion! We just had a short discussion with Jamie and we agree with:
- having a “Scala 3 Only” label at the top of the page
- maybe add a comment saying that the multiversal equality feature has been introduced in Scala 3 and has no equivalent in Scala 2, and all the code examples assume Scala 3
- no tab around the code blocks

Reply:
@julienrf Okay, that sounds good! I can do that later today, or on my Friday morning.